queries/rust/highlights.scm

```diff
@@ -41,9 +41,12 @@
     field: (field_identifier) @function))
 
 ; Assume other uppercase names are enum constructors
-([(identifier) (field_identifier)] @constant
+((field_identifier) @constant
```

Comment:
this query was making all identifiers that start with an uppercase letter constants. Not sure about the other node `(field_identifer)`